src/components/ToDoContainer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import ToDoList from './ToDoList';
+import Header from './Header';
+
+class TodoContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [
+        {
+          id: 1,
+          title: 'Learn React',
+          completed: false,
+        },
+        {
+          id: 2,
+          title: 'Learn Redux',
+          completed: false,
+        },
+        {
+          id: 3,
+          title: 'Learn React Router',
+          completed: false,
+        },
+      ],
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
Great job hardcoding the Todo List items. Please pass props from parent component to child components to allow for manipulation of state upon firing of the handleChange event.